src/components/transaction/DataFormat.vue

```diff
@@ -103,32 +115,41 @@ export default defineComponent({
             toggleOverflow,
             maxHeight,
             switchHeight,
-            useColor,
         };
     },
 });
 </script>
 
 <template>
-
-<div class="relative-position" :class="showOverflow ? '' : 'overflow-hidden'" :style=" (showOverflow || !isOverflowing) ? '' : `max-height: calc(${maxHeight}px - ${switchHeight}px)`">
-  <div class="row" v-if="actionName === 'transfer'" ref="dataBox">
-    <div class="col-12"><span class="text-bold">
-        <AccountFormat :account="transferData.from" type="account"></AccountFormat></span><span class="text-bold">&nbsp; → &nbsp;
-        <AccountFormat :account="transferData.to" type="account">&nbsp;</AccountFormat></span><span class="text-bold">{{ ' ' + transferData.quantity}}</span></div>
-    <div class="col-12">
-      <div class="text-weight-bold">memo:&nbsp;<span class="text-weight-regular" v-if="transferData.memo">{{transferData.memo}}</span><span class="text-weight-regular" v-else>n/a</span></div>
+<div
+    class="relative-position"
+    :class="showOverflow ? '' : 'overflow-hidden'"
+    :style="showOverflow || !isOverflowing ? '' : `max-height: calc(${maxHeight}px - ${switchHeight}px)`"
+>
```

Comment:
change in-line styling to a conditional class `:class=""{ overflow-height-class: !showOverflow && isOverflowing }"`

where (named arbitrarily for example) 
`.overflow-height-class {
      max-height: 37px //the calc values are hard-coded so don't need the variables
}`